inst/examples/build-a-box/R/mod-global-controls.R

```diff
@@ -0,0 +1,116 @@
+ui_global_controls <- function(id) {
+  ns <- shiny::NS(id)
+
+  tagList(
+    layout_columns(
+      class = "align-items-end",
+      selectizeInput(
+        ns("theme_style"),
+        "Theme style",
+        list(
+          "All" = "all",
+          "Semantic Colors" = list(
+            "Background" = "semantic-bg",
+            "Text" = "semantic-fg"
+          ),
+          "Theme Colors" = list(
+            "Background" = "colors-bg",
+            "Text" = "colors-fg"
+          ),
+          "Vibrant" = list(
+            "Gradient" = "gradient"
+          )
+        )
+      ),
+      div(
+        class = "btn-group",
+        role = "group",
+        "aria-label" = "Theme actions",
+        shuffleButton(ns("random_theme"), "Theme"),
+        shuffleButton(ns("random_stat"), "Stats"),
+        actionButton(ns("reset_theme"), "Reset")
+      )
```

Comment:
I think you could get away with not having this button group if you wanted. I'm not sure the Stats/Reset is needed and you could put a refresh icon in the theme dropdown (like is done on the other tabs)?

Reply:
I kind of like it. I've found it helpful to shuffle through the stats until I get a set that feel like what I'm looking for. And resetting is annoying to do. But I do think it makes more sense to have "Default" be a theme style; which helps the initial state of the input be more consistent with the value boxes (they start with default theme and the input says "default").